src/components/localization/languages.js

```diff
@@ -0,0 +1,75 @@
+
+import LocalizedStrings from 'react-localization';
```

Comment:
@Babiy7 seems to be a duplication of ukr.js file

Reply:
Deleted the file of the language.js which was duplicate for kor.js and ukr.js files.